biom/assets/exercise_api.py

```diff
@@ -0,0 +1,105 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2011-2015, The BIOM Format Development Team.
```

Comment:
2015 -> 2017

Probably better to use 'current' or leave it open so we don't have to change every year?